Fixed undo app name issue

S-Abdul-Rahman opened the pull request:
Fixed #7192

Ckbhatia commented on frontend/src/Editor/Header/EditAppName.jsx:
```diff
@@ -31,6 +31,19 @@ function EditAppName({ appId, appName = '', onNameChanged }) {
   };
 
+  React.useEffect(() => {
+    const handleKeyDown = (e) => {
```
You can move this function outside of this component

kavinvenkatachalam reviewed (changes requested):
@S-Abdul-Rahman We can call the function inside the useEffect and we need to remove the event listener when the component is unmounted

kavinvenkatachalam commented:
@arpitnath Will it be fixed in your PR?

S-Abdul-Rahman commented:
> @S-Abdul-Rahman We can call the function inside the useEffect and we need to remove the event listener when the component is unmounted

@kavinvenkatachalam  So, are you suggesting that these changes need to be reverted back to the previous ones? In the previous code, I added this function inside useEffect and I removed the event listener when the component unmounted.

arpitnath commented:
https://github.com/ToolJet/ToolJet/pull/7451#issuecomment-1735297159
@kavinvenkatachalam Yes
